python/docs/examples.rst

```diff
@@ -115,7 +119,7 @@ Here the only notable difference is:
 More on Kafka as the output connector at: https://www.feldera.com/docs/connectors/sinks/kafka
 
 .. warning::
-    Kafka is a streaming data source, therefore running: :meth:`.SQLContext.run_to_completion` will run forever.
+    Kafka is a streaming data source, therefore running: :meth:`.SQLContext.wait_for_completion` will block forever.
```

Comment:
Ideally, not this PR, an error should be thrown if a streaming data source is defined and the user tries to call `wait_for_completion()`.